pkg/abstractions/common/mount.go

```diff
@@ -3,31 +3,64 @@ package abstractions
 import (
 	"path"
 
+	"github.com/beam-cloud/beta9/pkg/common"
+	"github.com/beam-cloud/beta9/pkg/storage"
 	"github.com/beam-cloud/beta9/pkg/types"
 )
 
-func ConfigureContainerRequestMounts(stubObjectId string, workspaceName string, config types.StubConfigV1, stubId string) []types.Mount {
+const defaultExternalVolumesPath string = "/data/external-volumes"
```

Comment:
Shouldn't this have a different root path? If it's mounted to /data, won't it be a subpath of the juicefs mount?